host/class/cdc/usb_host_cdc_acm/test_app/main/test_app_main.c

```diff
@@ -7,17 +7,19 @@
 #include <stdio.h>
 #include <string.h>
 #include "unity.h"
-#include "esp_heap_caps.h"
+#include "unity_test_runner.h"
+#include "unity_test_utils_memory.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "usb/cdc_acm_host.h"
 
-static size_t before_free_8bit;
-static size_t before_free_32bit;
-
-#define TEST_MEMORY_LEAK_THRESHOLD (-530)
-static void check_leak(size_t before_free, size_t after_free, const char *type)
+void tearDown(void)
 {
-    ssize_t delta = after_free - before_free;
-    printf("MALLOC_CAP_%s: Before %u bytes free, After %u bytes free (delta %d)\n", type, before_free, after_free, delta);
-    TEST_ASSERT_MESSAGE(delta >= TEST_MEMORY_LEAK_THRESHOLD, "memory leak");
+    // Clean-up
+    ESP_ERROR_CHECK(cdc_acm_host_uninstall());
+    vTaskDelay(20); // Short delay to allow task to be cleaned up
```

Comment:
Let's keep `cdc_acm_host_install` and `uninstall` in the same layer, it the TEST_CASE.

It is unusual to install the driver in the TEST_CASE and uninstall it in the tearDown

Reply:
Now fixed